apps/admin-web/src/components/pages/tools/Events/Event.tsx

```diff
@@ -1,9 +1,9 @@
 import React, { setGlobal } from "reactn";
 
-import { useMutation } from "@apollo/react-hooks";
 import styled, { AnyStyledComponent } from "styled-components";
 
-import { DELETE_EVENT, GET_EVENTS } from "./helpers";
+import { GET_EVENTS } from "./helpers";
```

Comment:
Do you still need this import with the generated stuff?

Reply:
Yes, as I use it to refetch